src/lib/src/charts/chart.css

```diff
@@ -1,5 +1,5 @@
 :host > div {
-  background-color: var(--white);
+  background-color: white;
```

Comment:
var doesn't belong here, not til https://github.com/PRX/styleguide.prx.org/issues/42